packages/react-native/ReactAndroid/src/main/java/com/facebook/hermes/reactexecutor/HermesExecutorFactory.java

```diff
@@ -34,6 +35,7 @@ public void setDebuggerName(String debuggerName) {
     mDebuggerName = debuggerName;
   }
 
+  @NonNull
```

Comment:
JavaScriptExecutorFactory explicitly marks these values as non nullable, so it is ok to mark them here as well.
<img width="1840" alt="Screenshot 2024-12-24 at 23 27 06" src="https://github.com/user-attachments/assets/130764fd-3ab9-4aba-8065-4bcca3ce9bfc" />